engine/runtime-compiler/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -14,58 +14,32 @@ import org.enso.compiler.pass.resolve.MethodDefinitions
 import org.enso.editions.LibraryName
 import org.enso.pkg.{Config, QualifiedName}
 
-import java.io.ObjectOutputStream
-import scala.annotation.unused
 import scala.collection.mutable.ArrayBuffer
 
 /** A utility structure for resolving symbols in a given module.
```

Comment:
Add javadoc that this class is now thread safe?

Reply:
Adding slightly different note in 4a7c5a1 - it should advice people to focus on `BindingsMapBase` in the future. `BindingsMapBase` already contains a ["threading javadoc"](https://github.com/enso-org/enso/pull/13315#discussion_r2159434298).